src/main/resources/db/migration/V25__INSERT_BOOKS.sql

```diff
@@ -1,208 +0,0 @@
-BEGIN TRANSACTION;
```

Comment:
Instead of deleting this file, could you keep this in and a new migration (with a newer version number) for adding more books?

Reply:
Sure, will do